lib/craigslister.rb

```diff
@@ -1,9 +1,14 @@
 require 'nokogiri'
 require 'open-uri'
+require_relative './item_searcher'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.